docker-compose.yml

```diff
@@ -31,7 +31,7 @@ services:
       - "80:80"
       - "443:443"
     volumes:
-      - ./letsencrypt:/letsencrypt
+      - ${CONFIG_ROOT}/letsencrypt:/letsencrypt
```

Comment:
would you mind doing `${CONFIG_ROOT:.}` instead, so it defaults to `.` if the environment variable is not present in case somebody upgrades for example?

Reply:
Good idea, added.